packages/mask/src/resources/GemIcon.tsx

```diff
@@ -0,0 +1,82 @@
+import { SvgIconProps, SvgIcon } from '@mui/material'
+
+const svg = (
```

Comment:
Put it in `packages/icon/brands`. Suggest to put all these brand icons there.

Reply:
there are some icons not provided, added todo

Reply:
done